extensions/amp-story-education/0.1/amp-story-education.js

```diff
@@ -104,6 +118,23 @@ export class AmpStoryEducation extends AMP.BaseElement {
       true /** useCapture */
     );
 
+    // Prevent touchevents from being forwarded through viewer messaging.
```

Comment:
Should we remove these at some point?

Reply:
It's actually what has been discussed, the issue being that the second screen says "swipe" but if we don't prevent the touchevents from being forwarded, users would be sent to the next story. :(